app/facades/food_facade.rb

```diff
@@ -0,0 +1,5 @@
+class FoodFacade
+  def self.foods(food_input)
+    food_result = FoodService.return_food_query(food_input)
```

Comment:
This method is returning the entire JSON response to the controller, which will then send it to the view. There's A LOT in this JSON response that we don't need to send to the view-- there's a lot of food (of which we'll only select 10 of), and within each food there's a lot of data that we (at this moment) don't care to share with our client. This is where the PORO comes in.. let's create Food objects out of the first 10 foods, and THEN send that collection to the view.